SpatialGDK/Source/SpatialGDK/Private/Interop/GlobalStateManager.cpp

```diff
@@ -182,13 +182,16 @@ void UGlobalStateManager::ReceiveShutdownMultiProcessRequest()
 	}
 }
 
-void UGlobalStateManager::OnShutdownComponentUpdate(const Worker_ComponentUpdate& Update)
+void UGlobalStateManager::OnShutdownComponentUpdate(Schema_ComponentUpdate* Update)
 {
-	Schema_Object* EventsObject = Schema_GetComponentUpdateEvents(Update.schema_type);
+#if WITH_EDITOR
+	Schema_Object* EventsObject = Schema_GetComponentUpdateEvents(Update);
+	// TODO: Probably should be a bool in state - probably a non-persistent entity
```

Comment:
Should probably make a ticket for this.

Reply:
added https://improbableio.atlassian.net/browse/UNR-4395